1.8/development/create-package.md

```diff
@@ -0,0 +1,153 @@
+* The `preInstallNotes` parameter gives the user information they'll need before starting the installation process. For example, you could explain what the resource requirements are for the service: `"preInstallNotes": "Unicorns take 7 nodes with 1 core each and 1TB of ram."`
+* The `postInstallNotes` parameter gives the user information they'll need after the installation. Focus on providing a documentation URL, a tutorial, or both. For example: `"postInstallNotes": "Thank you for installing the Unicorn service.\n\n\tDocumentation: http://<service-url>\n\tIssues: https://github.com/"`
+* The `postUninstallNotes` parameter gives the user information they'll need after an uninstall. For example, further cleanup before reinstalling again and a link to the details. A common issue is cleaning up ZooKeeper entries. For example: `postUninstallNotes": "The Unicorn DC/OS Service has been uninstalled and will no longer run.\nPlease follow the instructions at http://<service-URL> to clean up any persisted state" }`
+
+See [`package.json`](https://github.com/mesosphere/universe#packagejson) for details on what can be defined in `package.json`.
+
+## Example `package.json`
+
+```json
+{
+  "packagingVersion": "2.0", // use either 2.0 or 3.0
+  "name": "foo", // your package name
+  "version": "1.2.3", // the version of the package
+  "tags": ["mesosphere", "framework"],
+  "maintainer": "help@bar.io", // who to contact for help
+  "description": "Does baz.", // description of package
+  "scm": "https://github.com/bar/foo.git", 
+  "website": "http://bar.io/foo", 
+  "framework": true,
+  "postInstallNotes": "Have fun foo-ing and baz-ing!"
+}
+```
+
+# `resource.json`
+
+This file declares all the externally hosted assets the package will need—for example: Docker containers, images, or native binary CLI.  See the [`resource.json`](https://github.com/mesosphere/universe#resourcejson) for details on what can be defined in resource.json.
+
+## Example `resource.json`
+
+```json
+{
+  "images": {
+    "icon-small": "http://some.org/foo/small.png",
+    "icon-medium": "http://some.org/foo/medium.png",
+    "icon-large": "http://some.org/foo/large.png",
+    "screenshots": [
+      "http://some.org/foo/screen-1.png",
+      "http://some.org/foo/screen-2.png"
+    ]
+  },
+  "assets": {
+    "uris": {
+      "log4j-properties": "http://some.org/foo/log4j.properties"
+    },
+    "container": {
+      "docker": {
+        "23b1cfe8e04a": "some-org/foo:1.0.0"
+      }
+    }
+  }
+}
+```
+
+# `config.json`
+
+This file declares the packages configuration properties, such as the amount of CPUs, number of instances, and allotted memory.  The defaults specified in `config.json` will be part of the context when `marathon.json.mustache` is rendered.
```

Comment:
I think it'd be good to include a note about config.json being a json schema and provide a link to the json schema docs.

In universe we have this as the leading sentence:
`This file describes the configuration properties supported by the package, represented as a
[json-schema](http://spacetelescope.github.io/understanding-json-schema/)`